package.json

```diff
@@ -27,7 +28,7 @@
   ],
   "homepage": "https://github.com/deltachat/deltachat-core-rust/tree/master/node",
   "license": "GPL-3.0-or-later",
-  "main": "node/dist/index.js",
+  "exports": "node/dist/index.js",
```

Comment:
this prevents desktop from building:
<img width="504" alt="Bildschirmfoto 2024-04-30 um 01 32 08" src="https://github.com/deltachat/deltachat-core-rust/assets/18725968/86eefd6c-6177-4a5c-83bb-a4cdb00f1d5a">


Though desktop should probably import the constants from the jsonrpc client instead anyway